tests/tests_default_vars.yml

```diff
@@ -7,7 +7,7 @@
         - import_role:
             name: nbde_server
         - assert:
-            that: "{{ item }} is defined"
+            that: item is defined
```

Comment:
I think `"{{ item }} is defined"` is correct.  I tried a simple playbook like this:
```yaml
- name: set parameters for test
  hosts: localhost
  tasks:
  - set_fact:
      astring: "astringval"
  - set_fact:
      bstring: "bstringval"
  - debug:
      msg: "{{ item }} is defined"
    loop:
      - astring
      - bstring
      - cstring
  - assert:
      that: item is defined
    loop:
      - astring
      - bstring
      - cstring
```
This incorrectly reported that cstring is defined.
```
ok: [localhost] => (item=cstring) => {
    "ansible_loop_var": "item",
    "changed": false,
    "item": "cstring",
    "msg": "All assertions passed"
}
```

Reply:
Heh, I did this one last, and it seemed fishy, but passed the test. Let me revert this portion.